index.js

```diff
@@ -10,7 +11,7 @@ var runtimes = [
   {
     name: 'RemoteDebug Core',
     protocol: require('./protocols/core/protocol.json'),
-    icon: 'RemoteDebug.png'
+    icon: 'remotedebug.png'
```

Comment:
ATM it's broken

<img width="98" alt="screen shot 2017-01-25 at 01 34 26" src="https://cloud.githubusercontent.com/assets/985504/22273049/73e9878e-e29e-11e6-98c4-de08374771ca.png">